docs/dev/installing.md

```diff
@@ -0,0 +1,260 @@
+
+Linux and macOS:
+```sh
+omz_downloader --name googlenet-v1 --output_dir ~/models
+```
+
+Windows:
+```bat
+omz_downloader --name googlenet-v1 --output_dir %USERPROFILE%\Documents\models
+```
+
+4. Convert the Model with Model Optimizer
+
+Linux and macOS:
+```sh
+mkdir ~/ir
+mo --input_model ~/models/public/googlenet-v1/googlenet-v1.caffemodel --data_type FP16 --output_dir ~/ir
+```
+Windows:
+```bat
+mkdir %USERPROFILE%\Documents\ir
+mo --input_model %USERPROFILE%\Documents\models\public\googlenet-v1\googlenet-v1.caffemodel --data_type FP16 --output_dir %USERPROFILE%\Documents\ir
+```
+
+5. Run Inference on the Sample
+
+Set up the OpenVINO environment variables:
+
+Linux and macOS:
+```sh
+source <INSTALLDIR>/setupvars.sh
+```
+
+Windows:
+```bat
+<INSTALLDIR>\setupvars.bat
+```
+
+The following commands run the Image Classification Code Sample using the [`dog.bmp`](https://storage.openvinotoolkit.org/data/test_data/images/224x224/dog.bmp) file as an input image, the model in IR format from the `ir` directory, and on different hardware devices:
+
+Linux and macOS:
+
+```sh
+cd ~/inference_engine_cpp_samples_build/intel64/Release
+./classification_sample_async -i ~/Downloads/dog.bmp -m ~/ir/googlenet-v1.xml -d CPU
+```
+
+Windows:
+
+```bat
+cd  %USERPROFILE%\Documents\Intel\OpenVINO\inference_engine_samples_build\intel64\Release
+.\classification_sample_async.exe -i %USERPROFILE%\Downloads\dog.bmp -m %USERPROFILE%\Documents\ir\googlenet-v1.xml -d CPU
+```
+
+When the sample application is complete, you see the label and confidence data for the top 10 categories on the display:
+
+```
+Top 10 results:
+
+Image dog.bmp
+
+classid probability
+------- -----------
+156     0.6875963
+215     0.0868125
+218     0.0784114
+212     0.0597296
+217     0.0212105
+219     0.0194193
+247     0.0086272
+157     0.0058511
+216     0.0057589
+154     0.0052615
+
+```
+
+</p>
+</details>
+
+## Adding OpenVINO Runtime (Inference Engine) to Your Project
+
+<details>
+<summary>For versions prior to 2022.1</summary>
+<p>
+
+For CMake projects, set the `InferenceEngine_DIR` and when you run CMake tool:
+
+```sh
+cmake -DInferenceEngine_DIR=/path/to/openvino/build/ .
+```
+
+Then you can find Inference Engine by [`find_package`]:
+
+```cmake
+find_package(InferenceEngine REQUIRED)
+target_link_libraries(${PROJECT_NAME} PRIVATE ${InferenceEngine_LIBRARIES})
+```
+</p>
+</details>
+
+
+<details open>
+<summary>For 2022.1 and after</summary>
+<p>
+
+
+For CMake projects, set the `OpenVINO_DIR` and when you run CMake tool:
+
+```sh
+cmake -DOpenVINO_DIR=<INSTALLDIR>/runtime/cmake .
+```
+
+Then you can find OpenVINO Runtime (Inference Engine) by [`find_package`]:
+
+```cmake
+find_package(OpenVINO REQUIRED)
+add_executable(ov_app main.cpp)
+target_link_libraries(ov_app PRIVATE openvino::runtime)
+
+add_executable(ov_c_app main.c)
+target_link_libraries(ov_c_app PRIVATE openvino::runtime::c)
+```
+</p>
+</details>
+
+## Additional Resources
```

Comment:
Empty section?
Please add a normal footer...

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2